code/tasks/SubsiteCopyPagesTask.php

```diff
@@ -12,7 +25,6 @@
 class SubsiteCopyPagesTask extends BuildTask
 {
     protected $title = 'Copy pages to different subsite';
-    
     protected $description = '';
```

Comment:
We might want to add `private static $segment = 'SubsiteCopyPagesTask';` otherwise it'll be namespaced, e.g. `sake dev/tasks/SilverStripe-Subsites-Tasks-SubsiteCopyPagesTask`